index.html

```diff
@@ -39,29 +39,234 @@ <h1 id="my-h1">I’m Crispin<br/> Glad to see<br/> you!</h1>
+                            </svg>
+                        </a>
+                    </li>
+                </ul><br/>
+                <div>
+                    <a href=""><button id="my-btn">Get my resume</button></a>
+                </div>
+                <div>
+                    <ul>
+                        <div class="about">
+                            <p class="paragraph">Languages</p>
+                            <svg width="24" height="24" viewBox="0 0 24 24" fill="none" xmlns="http://www.w3.org/2000/svg">
+                                <path fill-rule="evenodd" clip-rule="evenodd" d="M5.29289 8.29289C5.68342 7.90237 6.31658 7.90237 6.70711 8.29289L12 13.5858L17.2929 8.29289C17.6834 7.90237 18.3166 7.90237 18.7071 8.29289C19.0976 8.68342 19.0976 9.31658 18.7071 9.70711L12.7071 15.7071C12.3166 16.0976 11.6834 16.0976 11.2929 15.7071L5.29289 9.70711C4.90237 9.31658 4.90237 8.68342 5.29289 8.29289Z" fill="#5E6C84"/>
+                            </svg>
+                        </div>
+                        <li class="section6-list">
+                            <img src="icons/js icon.png" alt="js-icon"/>
+                            <p id="last-p">Javascript</p>
+                        </li><br/>
+                        <li class="section6-list">
+                            <img src="icons/html icon.png" alt="html-icon"/>
+                            <p id="last-p">HTML</p>
+                        </li><br/>
+                        <li class="section6-list">
+                            <img src="icons/css icon.png" alt="css-icon"/>
+                            <p id="last-p">CSS</p>
+                        </li>
+                    </ul>
+                </div>
+                <div>
+                    <ul>
+                        <li class="about">
+                            <p class="paragraph">Frameworks</p>
+                            <svg width="24" height="24" viewBox="0 0 24 24" fill="none" xmlns="http://www.w3.org/2000/svg">
+                                <path fill-rule="evenodd" clip-rule="evenodd" d="M5.29289 8.29289C5.68342 7.90237 6.31658 7.90237 6.70711 8.29289L12 13.5858L17.2929 8.29289C17.6834 7.90237 18.3166 7.90237 18.7071 8.29289C19.0976 8.68342 19.0976 9.31658 18.7071 9.70711L12.7071 15.7071C12.3166 16.0976 11.6834 16.0976 11.2929 15.7071L5.29289 9.70711C4.90237 9.31658 4.90237 8.68342 5.29289 8.29289Z" fill="#5E6C84"/>
+                            </svg>
+                        </li><hr/>
```

Comment:
- [ ] Kindly consider readjusting the color of this line.

Your design :point_down: 
![image](https://user-images.githubusercontent.com/97100091/205455208-d8b0e991-a51b-480b-bbe9-9e3509635748.png)

Figma design :point_down: 
![image](https://user-images.githubusercontent.com/97100091/205455189-e0c7546d-a69d-4dae-883a-e0d2c54faec2.png)